src/command_modules/azure-cli-network/azure/cli/command_modules/network/custom.py

```diff
@@ -3554,16 +3554,23 @@ def _set_route_table(ncf, resource_group_name, route_table, subnet):
 def create_subnet(cmd, resource_group_name, virtual_network_name, subnet_name,
                   address_prefix, network_security_group=None,
                   route_table=None, service_endpoints=None, service_endpoint_policy=None,
-                  delegations=None):
+                  delegations=None, nat_gateway=None):
```

Comment:
This must be gated behind the relevant API version or it will break on other profiles. What is the min API version that supports this?

Reply:
2019-02-01 updated in parameters